rllib/tests/test_worker_failures.py

```diff
@@ -410,9 +410,9 @@ def on_algorithm_init(self, *, algorithm, **kwargs):
                 algorithm.add_policy(
                     policy_id="test_policy",
                     policy_cls=PGTorchPolicy,
+                    config={},
```

Comment:
Wait, but then here we would need a complete config dict, correct? Not just this (empty) override.